Makefile

```diff
@@ -95,13 +125,25 @@ appstore:
 		-C $(sign_dir) $(app_name)
 	openssl dgst -sha512 -sign $(cert_dir)/$(app_name).key $(build_dir)/$(app_name).tar.gz | openssl base64
 
-test: deps
+test: build
 	./vendor/bin/phpunit --coverage-clover clover.xml -c phpunit.xml
 
 clean:
 	rm -rf ./build
-	rm -f js/handlebars.js js/lozad.js
-	rm -f models/1/mmod_human_face_detector.dat
-	rm -f models/1/dlib_face_recognition_resnet_model_v1.dat
-	rm -f models/1/shape_predictor_5_face_landmarks.dat
-	rm -r translationtool.phar
\ No newline at end of file
+	rm -f vendor/autoload.php
```

Comment:
can this be simplified with deleting everything from vendor, including it? Scary `rm -rf vendor`?:)

Reply:
We should not delete the files LICENCE.md and README.md of the models...